src/app/wallets/add-invoice-for-wallet.ts

```diff
@@ -102,12 +103,64 @@ const registerAndPersistInvoice = async ({
   return invoice
 }
 
-const walletIdFromWalletName = async (
-  walletName: WalletName,
+const walletIdFromPublicId = async (
+  walletPublicId: WalletPublicId,
+): Promise<WalletId | RepositoryError> => {
+  const walletsRepo = WalletsRepository()
+  const wallet = await walletsRepo.findByPublicId(walletPublicId)
+  if (wallet instanceof Error) return wallet
+
+  return wallet.id
+}
+
+// TODO: Remove. Used in v1 only
+const walletIdFromUsername = async (
+  username: Username,
 ): Promise<WalletId | RepositoryError> => {
   const walletsRepo = WalletsRepository()
-  const wallet = await walletsRepo.findByWalletName(walletName)
+  const wallet = await walletsRepo.findByUsername(username)
   if (wallet instanceof Error) return wallet
 
   return wallet.id
 }
+
+// TODO: Remove. Used in v1 only
+export const addInvoiceForUsername = async ({
+  username,
+  amount,
+  memo = "",
+}): Promise<LnInvoice | ApplicationError> => {
+  const walletPublicId = checkedToWalletPublicId(username)
```

Comment:
This feels a little odd (but may be ok in practice?).

On reading I would either expect `username` to be:
- checked user [`checkedToUsername`](https://github.com/GaloyMoney/galoy/pull/602/files#diff-c1ccf97093bb2237b9e83707b603c5c9594131c667291779378b17888e931a6dR10) method
- named/typed as `walletPublicId`/`WalletPublicId`
- be used in an intermediary step to retrieve the `walletPublicId` from the user and then used as `walletPublicId` from here

Reply:
Fixed